websites/D/Deezer/presence.ts

```diff
@@ -7,129 +7,164 @@ const presence = new Presence({
     live: "presence.activity.live"
   });
 
-function parseAudioTimestamps(
-  audioTime: string,
-  audioDuration: string
-): Array<number> {
-  const splitAudioTime = audioTime.split(":"),
-    splitAudioDuration = audioDuration.split(":"),
-    parsedAudioTime =
-      parseInt(splitAudioTime[0]) * 60 + parseInt(splitAudioTime[1]),
-    parsedAudioDuration =
-      parseInt(splitAudioDuration[0]) * 60 + parseInt(splitAudioDuration[1]),
-    startTime = Date.now(),
-    endTime =
-      Math.floor(startTime / 1000) - parsedAudioTime + parsedAudioDuration;
-  return [Math.floor(startTime / 1000), endTime];
+function parseTimestamp(time: string) {
+  return parseInt(time.split(":")[0]) * 60 + parseInt(time.split(":")[1]);
 }
```

Comment:
pretty sure there is a presence. function for this

Reply:
nope, it's a function made to specifically parse the audio timestamps since they're in a mm:ss format

Reply:
![image](https://user-images.githubusercontent.com/34792838/108362981-b2c2c580-71f4-11eb-8dd1-9ce356a883fb.png)

Reply:
presence.timestampFromFormat should be perfect for this.

Reply:
Alright will change it, thanks!